lua/telescope/builtin/init.lua

```diff
@@ -212,6 +212,8 @@ builtin.reloader = require('telescope.builtin.internal').reloader
 ---@field show_all_buffers boolean: if true, show all buffers, including unloaded buffers (default true)
 ---@field ignore_current_buffer boolean: if true, don't show the current buffer in the list (default false)
 ---@field only_cwd boolean: if true, only show buffers in the current working directory (default false)
+---@field sort_lastused boolean: if true, sort the shown buffers so that the last used one is highlighted (default false)
```

Comment:
one small nit. I think i prefer "selected" over "highlighted". Other LGTM thanks :)

Reply:
Well, that answers my question as well! :grin: